CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Because of the version bump to `go`, the macOS build for this release requires a
 1. [21839](https://github.com/influxdata/influxdb/pull/21839): Fix display and parsing of `influxd upgrade` CLI prompts in PowerShell.
 1. [21898](https://github.com/influxdata/influxdb/pull/21898): Removed unused `chronograf-migator` package & chronograf API service, and updated various "chronograf" references.
 1. [21919](https://github.com/influxdata/influxdb/pull/21919): Fix display and parsing of interactive `influx` CLI prompts in PowerShell.
+1. [21941](https://github.com/influxdata/influxdb/pull/#21941): Upgrade to golang-jwt 3.2.1.
```

Comment:
```suggestion
1. [21941](https://github.com/influxdata/influxdb/pull/21941): Upgrade to golang-jwt 3.2.1.
```

Reply:
Forgot to double-check the `CHANGELOG` URL since that usually trips people up with backports...looks like the link is correct but there's a small typo.

Reply:
Whoops! DONE